forgotPassword.js

```diff
@@ -0,0 +1,37 @@
+function onSendCodeClick() {
+    var user_email = document.getElementById("input_forgot_password_page").value;
```

Comment:
Change the variable type from var to let or const

Reply:
Done